client/views/actions/topic.tpl

```diff
@@ -1,8 +1,8 @@
 {{#if from}}
 	{{> ../user_name nick=from}}
-	has changed the topic to:
+	{{translate 'client.topic.has_changed'}}
 {{else}}
-	The topic is:
+	{{translate 'client.topic.topic_is'}}
 {{/if}}
 
 <span class="new-topic">{{{parse text}}}</span>
```

Comment:
The same issue with word ordering here as above: `{{user}} has changed the topic to {{topic}}` may not work with all languages

Reply:
here's where {{before}} {{end}} could work.
`{{before}} {{translate 'client.topic.topic_is'}} {{end}}`

In english

`before` = `The topic is:`
`end` = ` `